src/lightning_lite/strategies/fairscale.py

```diff
@@ -180,15 +178,6 @@ def register_strategies(cls, strategy_registry: Dict) -> None:
             description=cls.__class__.__name__,
         )
 
-    def _reinit_optimizers_with_oss(self, optimizers: List[Optimizer]) -> List["OSS"]:
```

Comment:
this was duplicated dead code from earlier refactor, the same function is already there below